test/blocks/table/table.test.js

```diff
@@ -111,5 +114,15 @@ describe('table and tablemetadata', () => {
       expect(tooltipHeading.childNodes.length).to.equal(2);
       expect(tooltipHeading.querySelector('.milo-tooltip, .icon-tooltip')).to.exist;
     });
+
+    it('should apply aria-label to all selects within .filters on mobile', async () => {
+      window.innerWidth = 375;
+      window.dispatchEvent(new Event('resize'));
+      await delay(500);
```

Comment:
We try to avoid any wait time in our unit tests as to not delay the whole suite. You can look up `await clock.runAllAsync();` in the codebase and that should provide a hint on how to wait for things to happen without impact on run time.